planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -596,7 +593,6 @@ void BehaviorPathPlannerNode::run()
 
   // for debug
   debug_avoidance_msg_array_publisher_->publish(bt_manager_->getAvoidanceDebugMsgArray());
-  publishDebugMarker(bt_manager_->getDebugMarkers());
```

Comment:
`getDebugMarkers()` function is no longer needed?

Reply:
I'll remove it :+1:

Reply:
Remove in https://github.com/autowarefoundation/autoware.universe/pull/1711/commits/8f72d46025e7a041eb772b239ad2e9c9aa5a0e30.